.gitignore

```diff
@@ -7,6 +7,8 @@ pom.xml.versionsBackup
 # gradle ignore
 .gradle
 build/
+out/
```

Comment:
i dont think it matters but i think it's coming from intellij missing some config - it should use build directory (also no need for `examples/**/out/` as this one should cover it